### PR TITLE
Add KisMac2 (0.3.4)

### PR DESCRIPTION
**Pull request by kai5263499**
### Checklist
- [X] The commit message includes the cask’s name and version.
- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [X] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/kismac2.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'kismac2' do
```
Comment (vitorgalvao) [inline Casks/kismac2.rb]: Remove the `2`. The version reference mentions version numbers are to be removed.

File: Casks/kismac2.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  version '0.3.4'
+  sha256 '62408ccc503c104ede70f8ad1ee55998a6c60ad89411fc2a57ce6be7bab39db5'
+
+  # downloads.igrsoft.com/beta/KisMac2.zip was verified as official when first introduced to the cask
+  url 'http://downloads.igrsoft.com/beta/KisMac2.zip'
+  name 'KisMAC2'
+  homepage 'https://github.com/IGRSoft/KisMac2'
```
Comment (vitorgalvao) [inline Casks/kismac2.rb]: Homepage is `https://igrsoft.com/kismac2/`.

File: Casks/kismac2.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask 'kismac2' do
+  version '0.3.4'
+  sha256 '62408ccc503c104ede70f8ad1ee55998a6c60ad89411fc2a57ce6be7bab39db5'
+
+  # downloads.igrsoft.com/beta/KisMac2.zip was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/kismac2.rb]: Remove the comment.

Comment (kai5263499) [inline Casks/kismac2.rb]: The style checker told me this comment was necessary when the url and homepage didn't match

File: Casks/kismac2.rb (hunk 2 above)
Comment (kai5263499) [inline Casks/kismac2.rb]: Is there another way to point to the source for the project? The igrsoft.com site is not as useful as linking to the source code

File: Casks/kismac2.rb (hunk 1 above)
Comment (kai5263499) [inline Casks/kismac2.rb]: I think the 2 is necessary since it's in the project name

File: Casks/kismac2.rb (hunk 3 above)
Comment (vitorgalvao) [inline Casks/kismac2.rb]: But [now they’ll match](https://github.com/caskroom/homebrew-cask/pull/24465/files/3f8473adc15d168cdf6bf61c884155fea8fe2c90#r78704422).

File: Casks/kismac2.rb (hunk 2 above)
Comment (vitorgalvao) [inline Casks/kismac2.rb]: The homepage clearly links to github.

File: Casks/kismac2.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/kismac2.rb]: It’s *inconsistently* on the project’s name. And the former one appears to be dead and this one is the new official one.

**Comment by kai5263499** [comment]
I've made the requested changes

**Comment by vitorgalvao** [comment]
Thank you.

**Comment by mikeyx2440** [comment]
Thank you  I appreciate your hard work

**Pull request merged**